gypbuild.js

```diff
@@ -22,9 +22,14 @@ function runGyp (opts, target, cb) {
     }
     args.push('--target=' + target)
     args.push('--target_arch=' + opts.arch)
-    if (opts.runtime === 'electron') {
-      args.push('--runtime=electron')
-      args.push('--dist-url=https://atom.io/download/electron')
+    switch (opts.runtime) {
```

Comment:
I personally prefer an `if-else` instead of a switch statement. Not sure what others think.